packages/client/src/Client.ts

```diff
@@ -111,8 +111,13 @@ export interface Client extends SharedClient<MinimalClient> {
     objs: Osdk<T>[],
     links: string[],
   ) => AsyncGenerator<BulkLinkResult, void, undefined>;
+
+  [additionalContext]: MinimalClient;
```

Comment:
Ok so we made a symbol internal to client here to hide stuff that isn't need for sharedClient

Reply:
exactly. And because its internal to the package we are not in trouble of symbol issues